rootfs/usr/share/inputplumber/devices/50-rog_ally.yaml

```diff
@@ -53,6 +53,9 @@ source_devices:
         x: [1, 0, 0]
         y: [0, -1, 0]
         z: [0, 0, -1]
+  - group: led
+    led:
+      name: ASUS ROG Ally Config
```

Comment:
We should also add these to the `rootfs/usr/share/inputplumber/schema/composite_device_v1.json` schema file.

Also, is "ASUS ROG Ally Config" the actual name returned by the driver?

Reply:
Yes, that's the name luke gave it